### PR TITLE
Homework/w1

### PR DESCRIPTION
**Pull request by crazymannnn**

### CODE REVIEW

File: Way Too Long Words/Program.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                if (arr[i].Length <= 10)
+                {
+                    Console.WriteLine(arr[i].ToLower());
+                }
+                if (arr[i].Length > 10)
+                {
+                    string first = arr[i][0].ToString();
+                    string last = arr[i][count - 1].ToString();
+                    Console.WriteLine(first.ToLower() + (count - 2) + last.ToLower());
+                }
```
Comment (HongDiep1806) [inline Way Too Long Words/Program.cs:28]: You can write a function to solve this task, your code in main function will be easier to see.

**Review by linlekms — APPROVED** [review]
LGTM -  (Look good to me)

File: DominoPiling/Program.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿namespace DominoPiling
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
+            int n = rd.NextInt();
+            int m = rd.NextInt();
```
Comment (linlekms) [inline DominoPiling/Program.cs:9]: you should use var to declare variables

File: DominoPiling/Program.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            Console.WriteLine((n * m) / 2);
+
+        }
+        class Reader
+        {
```
Comment (linlekms) [inline DominoPiling/Program.cs:14]: To increase readability, you should add a line to separate these 2 functions

File: Next Round/Program.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            Reader rd = new Reader();
+            int n = rd.NextInt();
+            int k = rd.NextInt();
+            int[] arr = new int[n];
+            for (int i = 0; i < n; i++)
+            {
+                arr[i] = rd.NextInt();
+            }
+            int aws = countOfPeo(k, arr);
+            Console.WriteLine(aws);
```
Comment (linlekms) [inline Next Round/Program.cs:16]: Use var to declare variables

File: Next Round/Program.cs [hunk 5]
```diff
@@ -0,0 +1,52 @@
+﻿namespace Next_Round
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
+            int n = rd.NextInt();
+            int k = rd.NextInt();
+            int[] arr = new int[n];
```
Comment (linlekms) [inline Next Round/Program.cs:10]: Use meaningfull names for variables

File: A. Team/Program.cs
Comment (HongDiep1806) [inline A. Team/Program.cs:1]: Look good to me

File: Next Round/Program.cs [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        }
+        public static int countOfPeo(int k, int[] arr) {
+            int count = 0;
+            for (int i = 0; i < arr.Length; i++) { 
+                if (arr[i] > 0 && arr[i] >= arr[k-1]) {
+                    count++;
+                }
+            }
+            return count;
+            }
```
Comment (linlekms) [inline Next Round/Program.cs:27]: Please format code

File: Next Round/Program.cs [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+            int[] arr = new int[n];
+            for (int i = 0; i < n; i++)
+            {
+                arr[i] = rd.NextInt();
+            }
+            int aws = countOfPeo(k, arr);
+            Console.WriteLine(aws);
+
+        }
+        public static int countOfPeo(int k, int[] arr) {
```
Comment (linlekms) [inline Next Round/Program.cs:19]: The function's name is not clear

**Comment by phamthienkinh** [comment]
LGTM

File: Petya and Strings/Program.cs [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
+            string s1 = rd.Next();
+            string s2 = rd.Next();
+            if (s1.ToLower() == s2.ToLower()) {
+                Console.WriteLine(0);
+            }
+            char[] ss1 = (s1.ToLower()).ToCharArray();
+            char[] ss2 = (s2.ToLower()).ToCharArray();
```
Comment (linlekms) [inline Petya and Strings/Program.cs:14]: follow naming convention and declaration as mentioned above

File: Boy or Girl/Program.cs [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+            char[] s1 = s.ToCharArray();
+            int count = 1;
+            for (int i = 1; i < s.Length; i++)
+            {
+                if (s1[0] != s1[i])
+                {
+                    count++;
+                }
+            }
+            int j;
```
Comment (HongDiep1806) [inline Boy or Girl/Program.cs:18]: I think u can write it directly in the 2nd loop

File: Way Too Long Words/Program.cs [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+                int count = arr[i].Length;
+                if (arr[i].Length <= 10)
+                {
+                    Console.WriteLine(arr[i].ToLower());
+                }
+                if (arr[i].Length > 10)
+                {
+                    string first = arr[i][0].ToString();
+                    string last = arr[i][count - 1].ToString();
+                    return (first.ToLower() + (count - 2) + last.ToLower());
```
Comment (linlekms) [inline Way Too Long Words/Program.cs:31]:
```suggestion
                    return $"{first.ToLower()}{count - 2}{ last.ToLower()}";
```

File: Way Too Long Words/Program.cs [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+            }
+            Console.WriteLine(OutPut( arr));
+        }
+        public static string OutPut(string[] arr) {
+            for (int i = 0; i < arr.Length; i++)
+            {
+                int count = arr[i].Length;
+                if (arr[i].Length <= 10)
+                {
+                    Console.WriteLine(arr[i].ToLower());
```
Comment (linlekms) [inline Way Too Long Words/Program.cs:25]: Try to use StringBuilder

**Review by linlekms — COMMENTED** [review]
Please take a look and improve your code

File: A. Team/Program.cs [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
+            int countOfSol = 0;
+            int count = 0;
+            int n = rd.NextInt();
+            int[, ] arr = new int[n,3];
```
Comment (linlekms) [inline A. Team/Program.cs:11]: Use var to declare variables
```suggestion
            var rd = new Reader();
            var countOfSol = 0;
            var count = 0;
            var n = rd.NextInt();
            var arr = new int[n,3];
```

File: A. Team/Program.cs [hunk 13]
```diff
@@ -0,0 +1,52 @@
+﻿namespace A._Team
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
```
Comment (linlekms) [inline A. Team/Program.cs:7]: Variables must have meaningful names
```suggestion
            var reader = new Reader();
```

File: A. Team/Program.cs (hunk 12 above)
Comment (linlekms) [inline A. Team/Program.cs:11]: Please double check whether we can name them as countOfSol, arr

**Pull request closed without merging**